app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def flash_class(level)
```

Comment:
For this method I added comment on branch with bootstrap. We can discuss this later.